packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec

```diff
@@ -27,54 +23,43 @@ BuildRequires: (rubygem(fog-proxmox) >= 0.15 with rubygem(fog-proxmox) < 1)
 BuildArch: noarch
 Provides: foreman-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
-%if 0%{?fedora} >= 27 || 0%{?rhel} >= 8
-Requires(post):   policycoreutils-python-utils
-Requires(postun): policycoreutils-python-utils
-%else
-Requires(post):   policycoreutils-python
-Requires(postun): policycoreutils-python
-%endif
```

Comment:
I think this should be kept
(but probably cleaned up, the else branch is useless now)